pkg/sensors/tracing/killer.go

```diff
@@ -20,20 +20,32 @@ import (
 	"github.com/cilium/tetragon/pkg/tracingpolicy"
 )
 
-type killerSensor struct{}
+const (
+	killerDataMapName = "killer_data"
+)
 
-func init() {
-	killer := &killerSensor{}
-	sensors.RegisterProbeType("killer", killer)
-	sensors.RegisterPolicyHandlerAtInit("killer", killerSensor{})
+type killerHandler struct {
+	configured   bool
+	syscallsSyms []string
+}
+
+func newKillerHandler() *killerHandler {
```

Comment:
the multi killer functionality is still coming later right?

Reply:
yap, still coming :)